target/linux/pistachio/base-files/etc/diag.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+. /lib/functions/leds.sh
+. /lib/pistachio.sh
+
+status_led="marduk:red:heartbeat"
+
+set_state() {
+
+	case "$1" in
+	preinit)
+		status_led_blink_preinit
+		;;
+	failsafe)
+		status_led_blink_failsafe
+		;;
+	preinit_regular)
+		status_led_blink_preinit_regular
+		;;
+	upgrade)
+		status_led_blink_slow
+		;;
+	done)
+		status_led_set_heartbeat
```

Comment:
I am happy to drop the use of status_led_blink_slow if it helps. However I'd quite like to keep the use of heartbeat:
* For info of lockup/panic, debatable but it can give you information of system status
* For consistency, the dts has been upstreamed to the kernel naming that led heartbeat and defining a default trigger of heartbeat
* There would be a lot of public documentation that would be inconsistent

If there are major concerns I can change this but believe there are valid reasons for keeping it. Also just because LEDE isn't using that function doesn't mean there aren't any forks that do make full use of it.